src/containers/collectibles/components/CollectiblesPage.tsx

```diff
@@ -394,15 +506,22 @@ const CollectiblesPage: React.FC<{
             </div>
           </div>
 
-          {isUserOnTheirProfile && (
-            <Button
-              type={ButtonType.COMMON}
-              size={ButtonSize.TINY}
-              text={collectibleMessages.edit}
-              leftIcon={<IconPencil />}
-              onClick={handleEditClick}
-            />
-          )}
+          <PopupMenu
+            items={overflowMenuItems}
+            // TODO: Bottom left is offset a bit. Need to look into that
```

Comment:
what should this be doing?

Reply:
When I set it to bottom left to match the designs, it looks like this so this was a note to look into the component to try to fix in stems
<img width="311" alt="Screen Shot 2021-11-16 at 2 40 40 PM" src="https://user-images.githubusercontent.com/23732287/142054255-de543966-55bb-44c3-a603-fc0839b5b5a9.png">

Reply:
gotcha 👍